index.html

```diff
@@ -0,0 +1,256 @@
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">CSS</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">JavScript</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">html</a>
+              </li>
+            </ul>
+
+            <div class="card_btn act_btn">
+              <a href="#" class="act_txt">See Project</a>
+            </div>
+          </div>
+        </article>
+
+        <article class="card">
+          <div class="card_img">&nbsp;</div>
+          <div class="card_details abs bottom_0">
+            <h1>
+              Multi-Post Stories <br />
+              Gain+Glory
+            </h1>
+
+            <ul class="tags">
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">Ruby on rails</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">CSS</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">JavScript</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">html</a>
+              </li>
+            </ul>
+
+            <div class="card_btn act_btn">
+              <a href="#" class="act_txt">See Project</a>
+            </div>
+          </div>
+        </article>
+
+        <article class="card">
+          <div class="card_img">&nbsp;</div>
+          <div class="card_details abs bottom_0">
+            <h1>
+              Multi-Post Stories <br />
+              Gain+Glory
+            </h1>
+
+            <ul class="tags">
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">Ruby on rails</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">CSS</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">JavScript</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">html</a>
+              </li>
+            </ul>
+
+            <div class="card_btn act_btn">
+              <a href="#" class="act_txt">See Project</a>
+            </div>
+          </div>
+        </article>
+      </div>
+    </section>
+    <section class="about section">
+      <section class="info">
+        <h1>About Me</h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="card_btn act_btn">
+          <p class="act_txt">Get My Resume</p>
+        </div>
+      </section>
+    </section>
+  </body>
+</html>
```

Comment:
- [ ] **Your website is missing the entire skills section. Please add it as it is a mandatory requirement towards project completion.**

Skill section in the original design
:-------------------------------------------:
![image](https://user-images.githubusercontent.com/59168430/120905232-4f0ebc00-c66e-11eb-98fd-04ac4ce5a9bb.png)